src/Extensions/Hosting.Services/Internal/ServiceContextAccessor.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Fabric;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services
+{
+	internal class ServiceContextAccessor<TContext> : IServiceContextAccessor<TContext>
```

Comment:
I've added comments to methods, but I still think it's a bit useless for the internal class since it's easier to look in code instead of comments that might go out or sync